### ISSUE TITLE
Upload progress is misleading

### ISSUE THREAD

**Issue by honzahana** [flowjs/flow.js#288]
Issue:

1. Uploading the first set of files is complete.
2. I will select additional files to upload.

In second step the output of `progress()` function is misleading. The function does not start counting the progress from zero. It also uses the previously uploaded files for calculation.

Is it possible to delete a file database of `Flow` after a successful upload of all files?

**Comment by AidasK** [comment]
Yes, it's sort of misleading, though it handles the case once you upload
some files and add more during the upload. Do you have a solution for this
issue? If it's simple, I could merge it.

**Comment by JonUK** [comment]
I achieved this manually by subscribing to the `fileSuccess` event, in this event checking if each file managed by Flow is complete `.isComplete()` or errored `.error` and if they are then removing each complete file one at a time with the method `.removeFile()`.

Removing the files once all of them are complete ensures that they then don't affect the overall progress when you next add new files. Depending on your use case, you may or may not want to remove files if there are failed uploads.
